### PR TITLE
Minor BugFixes, Add PlaceHolders for Training session students, Storage improved.

### PR DESCRIPTION
**Pull request by Whleee**
Fixed the Bug where Training Sessions are not stored after the programme exits.

Fixed cranky looking error messages.

Added Student placeholders for the example Training Session

### CODE REVIEW

File: src/main/java/seedu/address/model/student/Student.java [hunk 1]
```diff
@@ -226,4 +226,22 @@ public String toString() {
         getTags().forEach(builder::append);
         return builder.toString();
     }
+
+    /**
+     * Returns only the essentail information of Students (Name, Phone, Email, Academic Year).
```
Comment (sudogene) [inline src/main/java/seedu/address/model/student/Student.java]: Id included?

Comment (Whleee) [reply src/main/java/seedu/address/model/student/Student.java]: yup added, just the documentation never write, just edited it.

File: src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json [hunk 2]
```diff
@@ -23,5 +23,7 @@
     "thursdayDismissal" : "1500",
     "fridayDismissal" : "1500",
     "id": "1"
-  } ]
+  } ],
+
+  "trainings" : []
```
Comment (sudogene) [inline src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json:28]: We putting training data in student json file too?

**Review by sudogene — COMMENTED** [review]
Just some questions

File: src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json (hunk 2 above)
Comment (Whleee) [reply src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json:28]: cos AddressBook now needs to have a training field, just added it in so the test cases can run

Comment (yejiadong) [reply src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json:28]: @sudogene actually yes, each student will also store the dates of trainings assigned to them, so should be in their json too, we can look at this again after the add student to training command is added

**Review by yejiadong — APPROVED** [review]
LGTM.

**Pull request merged**